lua/nvim-tree/lib.lua

```diff
@@ -34,6 +34,8 @@ function M.init(with_open, with_reload)
 
   if with_open then
     M.open()
+  elseif (view.win_open()) then
```

Comment:
could you just remove the parens around the condition to keep the codestyle please ?